### DISCUSSION TITLE
What is the license for the Splashscreen artwork?

### DISCUSSION THREAD

**Discussion by matmanna** [Orama-Interactive/Pixelorama#753] [Q&A] [answered]
What is the license for the Pixelorama splash screen artwork? Is it Creative Commons or MIT licensed? I am specifically referring to this file:
https://github.com/Orama-Interactive/Pixelorama/blob/master/assets/graphics/splash_screen/artworks/roroto/roroto_004.png

An answer to this would be appreciated. Thanks! :)

**Comment by RorotoSic** [comment, marked as answer]
Hello, i can talk only for my artwork (the one of the link). It is CC-BY-NC-ND (= Creative Commons with Attribution, NonCommercial, NoDerivatives)

**Reply by matmanna** [reply]
@RorotoSic Thank you so much! I am guessing attribution can be included in a credits page. (of my project)

**Comment by OverloadedOrama** [comment]
Since the entire repository is licensed under MIT, I think that the artworks are technically also being licensed under MIT, unless it is stated otherwise, like we do with [addons](https://github.com/Orama-Interactive/Pixelorama/tree/master/addons). Question is, how should different licensed be stated? Should non-MIT artworks have a text file in the same directory as them in the repository with their license?

**Reply by matmanna** [reply]
@OverloadedOrama I think including a license in the artworks or splash screen directory would work... Otherwise, you could list the artists as "credits" in README.md along with the license for each of their respective creations...

**Reply by Variable-ind** [reply]
Since splashes change when a new major is released it may be appropriate to add it in SplashDialog.gd as one line comments e.g
#Made by RorotoSic licence CC-BY-NC-ND

**Reply by matmanna** [reply]
@Variable-ind That definitely seems like a valid solution!
